src/SeqCli/Config/SeqCliConnectionConfig.cs

```diff
@@ -23,6 +23,9 @@ class SeqCliConnectionConfig
     const string ProtectedDataPrefix = "pd.";
 
     public string ServerUrl { get; set; } = "http://localhost:5341";
+    public ulong EventBodyLimitBytes { get; set; } = 256 * 1024;
+    public ulong RawPayloadLimitBytes { get; set; } = 10 * 1024 * 1024;
```

Comment:
The `Raw*` in this name used to refer to the "raw" (as opposed to CLEF) output format. It might be better named just `PayloadLimitBytes` now, I think.